scss/_reboot.scss

```diff
@@ -94,13 +94,13 @@ body {
   @include focus-visible(); // 1
 }
 
-.js-focus-visible :focus:not([data-focus-visible-added]),
-.js-focus-visible .focus:not([data-focus-visible-added]) { // 2
+.js-focus-visible :focus:not([data-focus-visible-added]):not(.focus-ring),
+.js-focus-visible .focus:not([data-focus-visible-added]):not(.focus-ring) { // 2
   outline: 0 !important;
   box-shadow: none;
 }
 
-:focus:not(:focus-visible) { // 3
+:focus:not(:focus-visible):not(.focus-ring) { // 3
```

Comment:
Missing l.515 for buttons ?

Reply:
Added it in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1838/commits/4ee6759c022a86552e7dcad98de04237d02a32b0.
Please double-check that there's no impact on our buttons.